src/org/zaproxy/zap/extension/bruteforce/ZapAddOn.xml

```diff
@@ -6,6 +6,7 @@
 	<url></url>
 	<changes>
 	<![CDATA[
+	Adding request count on BruteForcePanel as specified on issue #1873
```

Comment:
Probably better to say "Forced Browse tab" instead of "BruteForcePanel" as this changes are shown to the user (which might not, immediately, associate with the tab).